### PR TITLE
#177903406 add notifications to services mail is sent

### PR DESCRIPTION
**Pull request by tobieeniafe**
#### What does this PR do?
Add notifications to services mail is sent

#### Description of Task to be completed?
Add notifications to services mail is sent

#### How should this be manually tested?
`npm run test`

#### What are the relevant pivotal tracker stories?
#177903406

### CODE REVIEW

File: server/services/nextPayment.service.js [hunk 1]
```diff
@@ -221,6 +223,14 @@ export const sendReminder = async () => {
     { $unwind: '$userInfo' },
     { $unwind: '$propertyInfo' },
   ]);
+
+  await Promise.all(
+    reminders.map(async (reminder) => {
+      await createNotification(NOTIFICATIONS.PAYMENT_REMINDER, reminder.userId);
```
Comment (andela-hpopoola) [inline server/services/nextPayment.service.js:229]: It looks like the services test for this is missing? Can we add it 🙏🏽

File: server/services/visitation.service.js [hunk 2]
```diff
@@ -156,6 +159,12 @@ export const processVisitation = async ({ user, visitationInfo, action }) => {
       { new: true },
     );
 
+    if (action === PROCESS_VISITATION_ACTION.RESCHEDULE) {
```
Comment (andela-hpopoola) [inline server/services/visitation.service.js:162]: I dont think we need to define another if statement here, we can add the notification to the first statement on line 149.

**Review by andela-hpopoola — APPROVED** [review]
Thank you for working on this. I left a few comments but overall, this looks good to me. It looks like we are still missing notifications for PAYMENTS since they are not part of the email we send. We can do that in a different PR though.

File: server/services/visitation.service.js (hunk 2 above)
Comment (tobieeniafe) [reply server/services/visitation.service.js:162]: I thought about this also, but I don't want us adding notifications, should the `findByIdAndUpdate` service fail and throw an error.

**Pull request merged**